content/docs/introduction/faq.md

```diff
@@ -60,8 +72,8 @@ have formal releases yet.
 
 Pulling over HTTP offers a number of advantages:
 
-* You can run your monitoring on your laptop when developing changes.
-* You can more easily tell if a target is down.
+* You can start extra monitoring instances as needed, e.g. on your laptop when developing changes.
+* You can more reliably tell if a target is down.
```

Comment:
I think "easily" was the correct word already - you can be just as reliable with push in telling whether a target is down, but it's harder because now you need to have configurations on both sides about target identities and what to expect.

Reply:
I was referring to Byzantine faults and near-time detection of failures; probably my ISP roots showing through.

Would "easily and reliably" be a consensus?

Reply:
SGTM